docker-compose.yml

```diff
@@ -84,6 +84,7 @@ services:
       - MSRC_KEYSTORE_FILE=/keystore/${KEYSTORE_FILE}
       - MSRC_PASSWORD=${PASSWORD}
       - MSRC_CHAIN_ID=${CHAIN_ID}
+      - MSRC_ACCEPT_DISCLAIMER=${MS_ACCEPT_DISCLAIMER}
```

Comment:
Is this `MS_ACCEPT_DISCLAIMER` intentionally (and not `MSRC_ACCEPT_DISCLAIMER`)? I don't know whether the disclaimers are different or need to be accepted separately for legal reasons.